src/components/StatePicker/index.tsx

```diff
@@ -23,11 +24,14 @@ type StatePickerProps = {
     /** Label to display on field */
     label?: string;
 
+    /** Any additional styles to apply */
+    wrapperStyle?: MenuItemProps['wrapperStyle'];
```

Comment:
NAB, but why not `MenuItemProps.wrapperStyle` here?

Reply:
MenuItemProps is a type, not a namespace ;)